templates/include/telemetry/histogram.1.schema.json

```diff
@@ -33,7 +33,13 @@
       "minimum": 0
     },
     "values": {
-      "type": "object"
+      "type": "object",
+      "patternProperties": {
+        "^[0-9].*$": {
+          "type": "string",
```

Comment:
The value-type should be `"integer"` here.

Reply:
Ah, this passed validation because I didn't set "additionalProperties" to false.